### DISCUSSION TITLE
Add HTTP Headers in pages

### DISCUSSION THREAD

**Discussion by aheissenberger** [vikejs/vike#697] [Discussion] [answered]
How can I add a caching header e.g. `Cache-Control: max-age=604800` to a page.

I only found a way to add a custom property to `pageContext` and add a method into the server process to handle this custom pageContext when creating the response.

**Comment by brillout** [comment, marked as answer]
Sounds good to me.

**Reply by aheissenberger** [reply]
I hoped there is a "standard" way ;-)

**Reply by brillout** [reply]
Also, you may want to use [a custom export](https://vite-plugin-ssr.com/exports).

```js
export cache = true
```

So that you can reuse your logic throughout your pages.
